app/components/Layout/Footer.jsx

```diff
@@ -34,21 +71,67 @@ class Footer extends React.Component {
        );
     }
 
-    render() {
+    checkNewVersionAvailable(){
+        let request = oneoff();
```

Comment:
You don't need this oneoff http request, just use fetch:

```
        fetch("https://api.github.com/repos/bitshares/bitshares-ui/releases/latest")
        .then(reply => reply.json()
            .then(json => {
                /* Check versions etc */
            })
        );
```

Reply:
@svk31 Fetch is not available at all in IE and is only available in newer versions of other browsers, which is why I did not use it

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API

Reply:
Fair enough, but we decided to stop supporting IE a long time ago so that's not really a concern. The GUI already relies on fetch in many places so if the browser being used doesn't support it then the GUI won't function properly anyway.

Reply:
Edge does support it though: http://caniuse.com/#search=Fetch

Reply:
@svk31 Will use fetch then